test/file1-1.tsx

```diff
@@ -0,0 +1,129 @@
+/** @jsxRuntime classic */
+/** @jsx jsx */
+import { jsx } from '@emotion/react';
+import { Collapse } from '@xsite-ui/common.collapse';
+import { Image } from '@xsite-ui/common.image';
+import { CtaButton } from '@xsite-ui/fintech.cta-button';
+import { PoweredByBankrate } from '@xsite-ui/fintech.icons/powered-by-bankrate';
+import { ProductIdentifier } from '@xsite-ui/fintech.product-identifier';
+import { OfferProduct, ProductTypes } from '@xsite-ui/fintech.shared-types';
+import { getMoneyString } from '@xsite-ui/fintech.utils';
+import React, { useState } from 'react';
+
+import { CollapseContent } from './collapse-content';
+import { COLLAPSE_BUTTON_TEXT } from './constants';
+import {
+  ArrowIcon,
+  CollapseButton,
+  CollapseButtonText,
```

Comment:
The use of the while loop with a condition that will never be true is unnecessary and should be removed.